README.md

```diff
@@ -173,7 +176,7 @@ language plpgsql;
 A [sample Go migration 00002_users_add_email.go file](./example/migrations-go/00002_rename_root.go) looks like:
 
 ```go
-package migrations
+package main
```

Comment:
This is golang requirement (https://tip.golang.org/pkg/plugin/).
[...] A plugin is a Go main package with exported functions [...]
Without this change, plugin do not compile correctly (resulting binary is not a valid ELF binary). Fortunately, this change does not break goose API.